app/controller.go

```diff
@@ -193,6 +197,9 @@ func (c *Controller) forward(w http.ResponseWriter, r *http.Request, user UserIn
 }
 
 func (c *Controller) createHost(w http.ResponseWriter, r *http.Request, user UserInfo) error {
+	if c.opsConfig.CreateHostDisabled {
+		return NewMethodNotAllowedError("Create host operation is disabled", nil)
+	}
```

Comment:
Given that this config value won't change while the service is running it might make more sense to check for it in the `Handler()` method. If it's disabled you can register a different handler that always returns 405. WDYT?

Reply:
good point, done!